File: codigos/1 - estrutura sequencial/1015 - Distancia Ente Dois Pontos/Main.clj
(defn distancia [x1 y1 x2 y2]
    (Math/sqrt (+ (Math/pow (- x2 x1) 2) (Math/pow (- y2 y1) 2))))

(defn main []
    (println (format "%.4f" (float (apply distancia (repeatedly 4 read))))))

(main)

File: codigos/1 - estrutura sequencial/1015 - Distancia Ente Dois Pontos/Main.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner prompt = new Scanner(System.in);
        double x1 = prompt.nextDouble();
        double y1 = prompt.nextDouble();
        double x2 = prompt.nextDouble();
        double y2 = prompt.nextDouble();
        double distancia = Math.sqrt(Math.pow(x2-x1, 2) + Math.pow(y2-y1, 2));
        System.out.println(String.format("%.4f", distancia));
    }
}